repository-data/webfiles/src/main/resources/site/freemarker/hee/macros/cta-card.ftl

```diff
@@ -3,7 +3,7 @@
 <#macro ctaCard ctaCard>
     <#if ctaCard??>
         <#assign openInNewWindow=false/>
-        <#if ctaCard.ctaCardContentBlock.ctaLink.document??>
+        <#if ctaCard.ctaCardContentBlock?? && ctaCard.ctaCardContentBlock.ctaLink.document??>

```

Comment:
Including the condition `ctaCard.ctaCardContentBlock??` on `<if>` makes it to move the control to `<else>` block and fails on `<#assign link = "${ctaCard.ctaCardContentBlock.ctaLink.url}">` (line number `12`)
```
[WARNING] [talledLocalContainer] %d{HH:mm:ss} WARN  [HstFreemarkerServlet] The following has evaluated to null or missing:
[INFO] [talledLocalContainer] ==> ctaCard.ctaCardContentBlock  [in template "webfile:/freemarker/hee/macros/cta-card.ftl" at line 12, column 36]
```
Could we instead wrap the whole if/else block with the following:

```
<#if ctaCard.ctaCardContentBlock??>
   ...
   ...
</#if>
```
Thanks!

Reply:
Thanks @smanivasag , updated as per requested